### ISSUE TITLE
Upgrade csv-parse

### ISSUE THREAD

**Issue by ZebraFlesh** [postmanlabs/newman#2123]
newman 4.5.4 appears to be pinned to version 4.0.1 of csv-parse. This results in a security vulnerability that is reported by when running `npm install newman`. The fix is to upgrade version 4.4.6 or later of csv-parse: https://www.npmjs.com/advisories/1171

**Comment by ethanbensonagc** [comment]
Yes, please do this. `npm audit fix` and `npm audit fix --force` do not fix the issue. :)

**Comment by UniVRsity** [comment]
how do you do that though?

**Comment by jstaormina** [comment]
Looks like there is a pull request that has been open since December to update this, with failing integration tests and merge conflicts.

**Comment by halfpastfouram** [comment]
> Looks like there is a pull request that has been open since December to update this, with failing integration tests and merge conflicts.

See https://github.com/postmanlabs/newman/pull/1812.

**Comment by codenirvana** [comment]
This has been fixed in Newman v4.5.5.
